src/test/java/firstSimpleJavaTasks/BubbleSortTests.java

```diff
@@ -0,0 +1,25 @@
+package firstSimpleJavaTasks;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class BubbleSortTests {
+
+    private int[] newIntArray = {9,-5,32,-62,192,-225,9};
```

Comment:
if you are not planning to change variable values within a class, make them final to avoid accidental miss-use.

Reply:
All unchangeable variables were covered with "final"